xdsl/dialects/llvm.py

```diff
@@ -48,6 +50,7 @@
 
 from xdsl.parser import Parser
 from xdsl.printer import Printer
+from xdsl.dialects import arith
```

Comment:
I am not 100% happy about this import as it makes the llvm dialect depend on the arith dialect, a dialect I would consider to be higher up in the compilations stack.

Reply:
AFAIU, you mainly use it to not have to rewrite the FastMathFlags. Personally, I feel the other way around is the cleaner way (i.e. llvm defines the flags and arith imports them). What do others think?